LaTeX/TeX.sublime-syntax

```diff
@@ -113,17 +163,76 @@ contexts:
         - include: general-commands
         - include: macro-braces
 
-  greeks:
-    - match: ((\\)(alpha|beta|gamma|delta|epsilon|varepsilon|zeta|eta|theta|vartheta|iota|gamma|kappa|lambda|mu|nu|xi|pi|varpi|rho|varrho|sigma|varsigma|tau|upsilon|phi|varphi|chi|psi|omega|Gamma|Delta|Theta|Lambda|Xi|Pi|Sigma|Upsilon|Phi|Psi|Omega))(?=\b|_)
+  math-builtin:
+    - include: math-greeks
+    - include: math-functions
+    - include: math-relations
+    - include: math-largeops
+    - include: math-binops
+    - include: math-symbols
+    - include: math-accents
+    - include: math-arrows
+    - include: math-delimiters
+
+  math-greeks:
+    - match: (\\)({{greeks}})(?=\b|_)
```

Comment:
As all variables are already groups `(?x: ...)` they don't need to be defined as capture group.

```suggestion
    - match: (\\){{greeks}}(?=\b|_)
```

Reply:
updated